doc/tutorial.txt

```diff
@@ -650,7 +674,7 @@ The generated events are of these main categories:
 
   - rtapp_loop: event=start thread_loop=0 phase=0 phase_loop=0
   - rtapp_loop: event=end thread_loop=0 phase=0 phase_loop=0
-  
+
```

Comment:
please don't mix typo and new feature in the same patch or PR

Reply:
Ah looks like my text editor stripped the lines, sorry about that